packages/dds/tree/src/feature-libraries/modular-schema/modularChangeTypes.ts

```diff
@@ -112,6 +116,14 @@ export interface NodeExistsConstraint {
 	violated: boolean;
 }
 
+/**
+ * A constraint that is violated whenever the edit with the constraint is rebased.
+ * Once violated, this constraint should remain violated.
```

Comment:
"should" made sense when describing what the implementation should do, but now that it's implemented, "should" just sound as though it's not guaranteed.
```suggestion
 * Once violated, this constraint will remain violated.
```